src/app/@core/utils/themes.service.ts

```diff
@@ -11,18 +12,25 @@ export class ThemesService {
             value: 'default',
             name: 'Light',
             description: 'Thème de base clair (texte sombre sur un fond clair)',
+            icon: 'sun-outline',
         },
         {
             value: 'dark',
             name: 'Dark',
             description: '',
+            icon: 'moon-outline',
         },
     ];
     savedTheme: string;
 
-    constructor(private themeService: NbThemeService, private userService: UserData, private authService: NbAuthService) { }
+    constructor(private themeService: NbThemeService, private userService: UserData, private authService: NbAuthService, private media: MediaMatcher) { }
 
     init() {
+
+        const browserTheme = this.media.matchMedia('(prefers-color-scheme: dark)') ? 'dark' : 'light'; // :"light "
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
        const browserTheme = this.media.matchMedia("(prefers-color-scheme: dark)") ? 'dark' : 'light'; // :"light "
```